dvc/scm/git/__init__.py

```diff
@@ -235,3 +235,52 @@ def belongs_to_scm(self, path):
 
     def get_tree(self, rev):
         return GitTree(self.git, rev)
+
+    def _get_diff_trees(self, a_ref, **kwargs):
```

Comment:
why don't we use a named parameter here ... like `b_ref = None`?

Reply:
At first I thought, that maybe later I would have much more parameters in that functions :+1: and later I just forgot about that thing :)